Source/Renderer/UniformState.js

```diff
@@ -857,6 +857,23 @@ define([
     }
 
     /**
+     * Synchronizes the frustum's state with the camera state.  This is called
+     * by the {@link Scene} when rendering to ensure that automatic GLSL uniforms
+     * are set to the right value.
+     *
+     * @param {Object} camera The camera to synchronize with.
+     */
+    UniformState.prototype.updateCamera = function(camera) {
```

Comment:
Does this need to set any other matrices with `view` in them, e.g., MVP, or set any dirty flags?